docs/WebUI.md

```diff
@@ -0,0 +1,73 @@
+# WebUI
+
+NNI is a research platform for metalearning. It provides easy-to-use interface so that you could perform neural architecture search, hyperparameter optimization and optimizer design for your own problems and models.
+Web UI allows user to monitor the status of the NNI system using a graphical interface.
+
+## Deployment
+
+### To start the webui
+
+* cd webui
```

Comment:
webui is shipped within NNI package and is started by nnictl command.  we do not need to expose this yarn start method for end user.